### ISSUE TITLE
"Registry Certificate" button missing

### ISSUE THREAD

**Issue by DasFaultier** [goharbor/harbor#18744]
**Expected behavior and actual behavior:**

_Expected:_ The documentation at https://goharbor.io/docs/2.8.0/working-with-projects/working-with-images/pulling-pushing-images/#download-the-harbor-certificate states that users can download the Registry Certificate to use it on their workstations when pushing images to Harbor. I expected this button to be there, because it's in the documentation and I remember using it in older  Harbor Versions (as included in vSphere integrated containers 1.5), ...

_Actual:_ ... but the button isn't there. As this would be helpful for user acceptance of the repository, I'd like to see this again in future releases, if this wasn't intended behaviour (in which case, documentation should be updated).

Screnshot:
![Harbor_actual](https://github.com/goharbor/harbor/assets/1090236/16244628-ef2f-41b0-b3ff-a1927e60b50c)

**Steps to reproduce the problem:**

- Navigate to "Projects -> SomeProject"
- Try to find the "Registry Certificate" button next to the "Pull Command" Button

**Versions:**

- harbor version: 2.7.1
- docker engine version: 20.10.17, build 100c701
- docker-compose version: 1.25.0

**Comment by AllForNothing** [comment]
Does `/data/ca_download/ca.crt` exit in the server where your Harbor is installed? The UI renders this button according to whether there is this certificate.

**Comment by DasFaultier** [comment]
Hi @AllForNothing,

thank you for your help. My Harbor instance is not installed in `/data`, but in `/opt/harbor/`. In there, I found the directory `/opt/harbor/data/ca_download/` which did not contain the `ca.crt`. When I copied the certificate to that directory, Harbor instantly rendered the button as expected, without a restart. This solves my initial issue, though a docu update would be nice.

Thanks again,
Jörg
